src/lib/components/TransactionView.svelte

```diff
@@ -5,62 +5,87 @@
     export let category: string;
 </script>
 
-<div class="flex justify-center items-center mt-20 mb-20">
-    <div class="join-vertical">
-        <div class="flex items-left">
-            <!-- Adding the 'flex' and 'items-left' classes here -->
-            <div>ID:</div>
-            <div class="join">
-                <div>
-                    <input
-                        class="input input-bordered join-item"
-                        placeholder={id}
-                        value={id}
-                    />
+<div class="max-w-md mx-auto px-6 py-3 my-8 border border-custom">
+    <div class="flex max-w-md flex-line mx-auto pl-3 pr-1 pt-4 pb-4">
+        <div class="flex flex-col w-32 text-xl text-start mt-3 mr-8">
+            <div class="mb-6">ID</div>
+            <div class="mb-6">Date</div>
+            <div class="mb-6">Amount</div>
+            <div>Category</div>
+        </div>
+        <div class="join-vertical space-y-1">
+            <div class="flex items-left">
+                <!-- Adding the 'flex' and 'items-left' classes here -->
+                <!-- <div>ID:</div> -->
+                <div class="join">
+                    <div>
+                        <input
+                            class="input input-bordered join-item"
+                            placeholder={id}
+                            value={id}
+                        />
+                    </div>
                 </div>
             </div>
-        </div>
 
-        <div class="flex items-left">
-            <!-- Adding the 'flex' and 'items-left' classes here -->
-            <div>Date:</div>
-            <div>
+            <div class="flex items-left">
+                <!-- Adding the 'flex' and 'items-left' classes here -->
+                <!-- <div>Date:</div> -->
```

Comment:
Commenting this tag looks not reliable to me 🤔

Reply:
This comment also does not look reliable to me